gui/petitions.lua

```diff
@@ -132,6 +135,17 @@ local function getAgreementDetails(a)
             e_descr[#e_descr+1] = table.concat{"The ", df.historical_entity_type[e.type], " ", dfhack.TranslateName(e.name, true)}
             if we == e_id then our = true end
         end
+        for _, hf_id in ipairs(p.histfig_ids) do
+            local hf = df.global.world.history.figures[hf_id]
+            local race = df.creature_raw.find(hf.race)
+            local civ = df.historical_entity.find(hf.civ_id)
+            e_descr[#e_descr+1] = table.concat{"The ", race.creature_id, " ", df.profession[hf.profession],
+                                " ", dfhack.TranslateName(hf.name, true),
+                                NEWLINE,
+                                "of ", dfhack.TranslateName( civ.name , true )
+                            }
```

Comment:
```suggestion
            e_descr[#e_descr+1] = ('The %s %s %s\nof %s'):format(race.creature_id,
                            df.profession[hf.profession],
                            dfhack.TranslateName(hf.name, true),
                            dfhack.TranslateName(civ.name, true))
```

Reply:
I fixed the whitespaces and changed the formatting of this code block somewhat - I'm still not sure what form is more readable, `table.concat` or `string.format`.

There is no reason to change the changelog, I think -- `gui/petitions` is in the `New Scripts` block.

Reply:
:+1: